src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16581,6 +16581,31 @@ namespace UnitarySystems {
         return airNode;
     }
 
+    int getZoneEqIndex(
+        EnergyPlusData &state, std::string const &UnitarySysName, DataZoneEquipment::ZoneEquipType zoneEquipType, bool &errFlag, int const OAUnitNum)
+    {
+
+        if (state.dataUnitarySystems->getInputOnceFlag) {
+            UnitarySystems::UnitarySys::getUnitarySystemInput(state, UnitarySysName, false, OAUnitNum);
```

Comment:
The zone equipment flag should be true

    UnitarySystems::UnitarySys::getUnitarySystemInput(state, UnitarySysName, true, OAUnitNum);

Reply:
@rraustad I am going to make a change.